src/intrinsic/mod.rs

```diff
@@ -308,6 +307,8 @@ impl<'a, 'gcc, 'tcx> IntrinsicCallMethods<'tcx> for Builder<'a, 'gcc, 'tcx> {
                 let b = args[1].immediate();
                 if layout.size().bytes() == 0 {
                     self.const_bool(true)
+                    // The else if an immediate neighbor of this block.
+                    // It is moved here to comply to Clippy.
```

Comment:
Please remove this comment.

Reply:
Done in tempdragon@3b01fba0f5ecfd7e929fcf1948945ae75479c38d